apps/src/templates/progress/progressTypes.js

```diff
@@ -158,9 +158,23 @@ export const studentLessonProgressType = PropTypes.shape({
  * @property {array} bigQuestion
  * @property {string} description
  */
-export const lessonGroupType = PropTypes.shape({
+const lessonGroupShape = {
   id: PropTypes.number,
   displayName: PropTypes.string,
   bigQuestions: PropTypes.string,
   description: PropTypes.string
+};
+
+/**
+ * @typedef {Object} GroupedLesson
```

Comment:
I'm not sure about this name, especially since there's also a type called lessonGroup.  Is this type basically a lesson group with all of its child levels?  If so, when would we use just the "plain" lessonGroup above?  (My first thought was that maybe _this_ should be the type called lessonGroup?  Or maybe lessonGroupWithLevels?

Longer-term, I think we want to get away from these "nested" types but we still need to discuss that.

Reply:
i added some comments and renamed to `groupedLessonsType`. let me know if those changes make it clear enough.

i 100% agree about moving away from these nested types, and that it will require some discussion.